src/lib/chat/matrix/types.ts

```diff
@@ -15,3 +15,7 @@ export enum MatrixConstants {
   NEW_CONTENT = 'm.new_content',
   REPLACE = 'm.replace',
 }
+
+export enum CustomEventType {
+  SEND_ADMIN_MESSAGE = 'm.custom.send_admin_message',
```

Comment:
Is the `m.custom` a standard prefix?

`admin_message` is probably a better event name. "send" sounds like an action that's being performed. Eg: The matrix event is "EventType.RoomMessage" not "EventType.SendRoomMessage"

Reply:
This was just something I made up, not a standard prefix. Updated to `admin_message`.